src/hooks/use-transition-pipe.js

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { pipe, subscribe, makeSubject } from 'wonka';
+
+const TransitionPipeContext = React.createContext({});
+
+function TransitionPipeProvider(props) {
+  const [transitionSource, runTransition] = React.useRef(makeSubject()).current;
+  pipe(
+    transitionSource,
+    subscribe(transition => transition.start())
```

Comment:
Small nit 😅  we're never unsubscribing here, so if this provider ever updates you'll see duplicate transitions potentially.

Maybe it's worth just wrapping the entire subject and subscription in a useMemo with no deps? We won't have to unsubscribe since no consumers can access runSubscription if this unmounts